string topic
---
float64 x
float64 y
float64 theta
float64 dist
